pom.xml

```diff
@@ -318,57 +358,94 @@
+                                        </artifact>
                                     </test>
                                 </tests>
                             </member>
                             <member>
                                 <name>Debezium-Outbox</name>
                                 <dependencyManagement>
-                                    <dependency>io.debezium:debezium-quarkus-outbox:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-quarkus-outbox-deployment:${debezium-quarkus-outbox.version}</dependency>
+                                    <dependency>io.debezium:debezium-quarkus-outbox:${debezium-quarkus-outbox.version}
+                                    </dependency>
+                                    <dependency>
+                                        io.debezium:debezium-quarkus-outbox-deployment:${debezium-quarkus-outbox.version}
+                                    </dependency>
                                     <dependency>io.debezium:debezium-api:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-cassandra:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-db2:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-mongodb:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-mysql:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-oracle:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-postgres:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-sqlserver:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-connector-vitess:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-core:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-ddl-parser:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-embedded:${debezium-quarkus-outbox.version}</dependency>
-                                    <dependency>io.debezium:debezium-scripting:${debezium-quarkus-outbox.version}</dependency>
+                                    <dependency>
+                                        io.debezium:debezium-connector-cassandra:${debezium-quarkus-outbox.version}
+                                    </dependency>
```

Comment:
@adutra is this an editor? Could you please undo the reformatting? The only meaningful change the Cassandra client version property value, right?

Reply:
Oh my bad, I thought these cosmetic changes were made by `mvn clean process-resources` but it turns out I've been a victim of IntelliJ's new "format on save" feature. Let me redo everything again. I will force-push if you don't mind.